daft/io/_lance.py

```diff
@@ -40,12 +23,15 @@ def read_lance(
     index_cache_size: Optional[int] = None,
     default_scan_options: Optional[dict[str, str]] = None,
     metadata_cache_size_bytes: Optional[int] = None,
+    use_scalar_index: bool = False
 ) -> DataFrame:
     """Create a DataFrame from a LanceDB table.
 
     Args:
         url: URL to the LanceDB table (supports remote URLs to object stores such as `s3://` or `gs://`)
         io_config: A custom IOConfig to use when accessing LanceDB data. Defaults to None.
+        version: Optional version of the LanceDB table to read. If not specified, the latest version will be used.
```

Comment:
If you happen to know what the other parameters are for, could you add them for consistency